packaging/src/docker/entrypoint.sh

```diff
@@ -31,11 +31,19 @@ function initialize_hive {
   fi
   $HIVE_HOME/bin/schematool -dbType $DB_DRIVER $COMMAND $VERBOSE_MODE
   if [ $? -eq 0 ]; then
-    echo "Initialized schema successfully.."
+    echo "Initialized Hive Metastore Server schema successfully.."
   else
-    echo "Schema initialization failed!"
+    echo "Hive Metastore Server schema initialization failed!"
     exit 1
   fi
+  if [[ "${SERVICE_NAME:-}" == "hiveserver2" ]]; then
+      $HIVE_HOME/bin/schematool $COMMAND -dbType hive -metaDbType $DB_DRIVER -url jdbc:hive2://localhost:10000/default $VERBOSE_MODE
+      if [ $? -ne 0 ]; then
+        echo "Hive-Server 2 schema initialization failed!"
```

Comment:
nit: `Hive-Server 2` -> `HiveServer2`

Reply:
- Should I update https://hive.apache.org/ to use `Hive-Server 2 (HS2)`? But I don't know where the corresponding documentation is on the homepage.
- ![image](https://github.com/user-attachments/assets/e4bde5fd-94ba-4592-b6a3-843c874ea61f)

Reply:
I would prefer the HiveServer2, the documentation is here: https://github.com/apache/hive-site/blob/main/themes/hive/layouts/partials/features.html#L40

Reply:
- I submitted a new commit to revert the changes. I hadn't noticed the `javax.jdo.option.**` before.